pkg/util/util.go

```diff
@@ -105,9 +105,9 @@ func StrToInt32(val string) *int32 {
 	return n32
 }
 
-// Hash retrieves an encoded string using the md5 encrypt
+// Hash retrieves an encoded string using the sha1 encrypt
 func Hash(s string) string {
-	hasher := md5.New()
+	hasher := sha256.New()
```

Comment:
This cannot be changed. As the md5 hashes are generated and stored in the etcd database. If we change it, we wont be able to map it with old blockdevice records.